packages/client/components/GroupingKanban.tsx

```diff
@@ -174,6 +177,7 @@ export default createFragmentContainer(GroupingKanban, {
           id
         }
       }
+      spotlightSearchQuery
```

Comment:
+1 this seems like an unrelated fix

Reply:
We need `spotlightSearchQuery` here and in `DraggableReflectionCard` so that we have access to the query string in `useDraggableReflectionCard` and the search bar. Thanks for the review - now that we've had 2 reviewers, I'll merge the PR